cmd/platform/ldap.go

```diff
@@ -32,10 +33,11 @@ func ldapSyncCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if ldapI := einterfaces.GetLdapInterface(); ldapI != nil {
-		if err := ldapI.Syncronize(); err != nil {
-			CommandPrintErrorln("ERROR: AD/LDAP Synchronization Failed")
+		job, err := ldapI.StartSynchronizeJob(true)
+		if err != nil || job.Status == model.JOB_STATUS_ERROR {
+			CommandPrintErrorln("ERROR: AD/LDAP synchronization please check the server logs")
 		} else {
-			CommandPrettyPrintln("SUCCESS: AD/LDAP Synchronization Complete")
+			CommandPrettyPrintln("SUCCESS: AD/LDAP synchronization")
```

Comment:
any particular reason to remove the "complete" from the end of this?

Reply:
hah, No good reason, I deleted it in an earlier PR and didn't add it back until later.  I can make it the original string.